src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navbar: React.FC = () => (
+  <>
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            className={({ isActive }) => classNames('navbar-item',
+              {
+                'has-background-grey-lighter': isActive,
+              })}
+            to="/"
+          >
+            Home
+          </NavLink>
```

Comment:
Better to create a separate component for this NavLink and style it there. Than you could just use it here twice, and set props with name and route